draft-bbs-signatures.md

```diff
@@ -514,31 +524,25 @@ Procedure:
 
 2. (i1, i2, ..., iR) = RIndxs
 
-3. (A', Abar, d, e^, r2^, r3^, s^, c, (m^[i1],...,m^[iR])) = spk
-
-4. (w, h0, h[1],...,h[L]) = PK
-
-5. if A' == 1 return INVALID
-
-6. T1 = Abar - d
-
-7. T2 = P1 + h[i1] * Rmsg[1] + ... + h[iR] * Rmsg[R]
+3. (A', Abar, D, c, e^, r2^, r3^, s^, (m^[i1],...,m^[iR])) = spk
 
-8. Y1 = A' * e^ + h0 * r2^ + T1 * c
+4. (W, H0, H[1],...,H[L]) = PK
 
-9. Y2 = d * r3^ + h0 * s^ + h[i1] * m^[i1] + ... + h[iR] * m^[iR] - T2 * c
+5. C1 = (Abar - D) * c + A' * e^ + H0 * r2^ 
 
-10. c_v = H(Abar || A' || h0 || Y1 || d || h0 || h[i1] || ... || h[iR] || Y2 || nonce)
+6. T = P1 + H[i1] * Rmsg[i1] + ... H[iR] * Rmsg[R]
 
-11. if c != c_v return INVALID
+7. C2 = T * c + D * (-r3^) + H0 * s^ + H[i1] * m^[i1] + ... + H[iR] * m^[iR]
```

Comment:
Yes it holds. I have code to prove it

Reply:
Great! I would be interested seen the code. Regardless I would also be more at ease if we could prove correctness with math since this PR does not use exactly the formulas from the academic paper (i.e., `s’ = s + r2*r2` instead of `s’ = s - r2*r2` which is used in the paper etc.). But I may be missing something and the code could help.